src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Common/Microsoft.Azure.Search.Common.csproj

```diff
@@ -6,7 +6,7 @@
     <Description>Common types needed by the Azure Search .NET libraries. This is not the package you are looking for; It is only meant to be used as a dependency.</Description>
     <AssemblyTitle>Microsoft Azure Search Common Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Search.Common</AssemblyName>
-    <Version>5.0.3</Version>
+    <Version>9.0.0</Version>
```

Comment:
Is there a reason you are jumping from 5 to 9 in the version?

Reply:
Yes. In the search-preview branch, we've already had 6.0-preview, 7.0-preview, and today 8.0-preview releases. It was necessary to bump the major version for each of those releases due to breaking changes.

We're considering making our preview SDK a separate package so we don't burn through version numbers so quickly (among other reasons).

Reply:
Generally when something is a preview it is OK to make breaking changes. It is when it become stable (i.e. no prelease label) that it is important to bump major version when you make breaking changes.  Take a look at https://github.com/Azure/azure-sdk/blob/master/docs/engineering-system/releases.md#package-versioning. When doing multiple previews generally you would bump a number after the preview like 6.0.0-preview1, 6.0.0-preview2, etc.

Reply:
Also I should point out that while semver says it is ok to make breaking changes across major versions we generally try to avoid that in all but the most extreme cases because it causes difficult for our developers, especially in the .NET library ecosystem. 

cc @KrzysztofCwalina

Reply:
We generally don't do this lightly. This particular version represents a fundamental change in how our SDK is implemented. We don't anticipate changes of similar scope in the future. Also, unlike say ClientRuntime or JSON.NET, our SDK should be pretty high in the stack and so shouldn't suffer from a lot of the same dependency management problems.

Reply:
Also, regarding the prerelease versioning -- our preview SDKs have historically been very long-lived, and we try to version them with the same care as the GA SDKs. This is because we've had features that have remained in preview for years. Not ideal, but it is the reality for us right now.